webtau-rest/src/main/java/com/twosigma/webtau/http/HttpDocumentation.java

```diff
@@ -38,14 +38,19 @@ public void capture(String artifactName) {
                     requestHeader);
         }
 
-        if (lastValidationResult.getRequestType() != null && !lastValidationResult.isRequestBinary()) {
+        if (lastValidationResult.getRequestType() != null
+                && !lastValidationResult.isRequestBinary()
+                && lastValidationResult.getRequestContent() != null
+                && !lastValidationResult.getRequestContent().isEmpty()) {
             String fileName = "request." + fileExtensionForType(lastValidationResult.getRequestType());
             FileUtils.writeTextContent(path.resolve(fileName),
                     prettyPrintContent(lastValidationResult.getRequestType(),
                             lastValidationResult.getRequestContent()));
         }
 
-        if (lastValidationResult.getResponseType() != null) {
+        if (lastValidationResult.getResponseType() != null
+                && lastValidationResult.getResponseTextContent() != null
```

Comment:
since it is a 3rd time we need it, could we please add a method to StringUtils similar to apache utils to check for null or empty at once.